test/acceptance/features/steps/servicebindingoperator.py

```diff
@@ -14,34 +14,27 @@ def __init__(self,  name="service-binding-operator", namespace="openshift-operat
         self.namespace = namespace
         self.name = name
 
-    def check_resources(self):
+    def check_crd(self):
         crd_name = "servicebinding"
         assert self.openshift.is_resource_in(crd_name), f"CRD '{crd_name}' does not exist"
-
-        name_pattern = self.get_name_pattern()
-
-        output = self.openshift.search_resource_in_namespace("rolebindings", name_pattern, self.namespace)
-        assert output is not None, f"Unable to find role binding with name that maches '{name_pattern}' in namespace '{self.namespace}': {output}"
-
-        output = self.openshift.search_resource_in_namespace("roles", name_pattern, self.namespace)
-        assert output is not None, f"Unable to find role with name that maches '{name_pattern}' in namespace '{self.namespace}': {output}"
-
-        output = self.openshift.search_resource_in_namespace("serviceaccounts", name_pattern, self.namespace)
-        assert output is not None,  f"Unable to find service account with name that maches '{name_pattern}' in namespace '{self.namespace}': {output}"
-
         return True
 
+    def check_deployment(self):
+        sbo_namespace = self.openshift.search_namespace_for_resource("deployments", "service-binding-operator")
+        return sbo_namespace is not None and sbo_namespace != ""
```

Comment:
`search_namespace_for_resource` return only non-empty string, otherwise, assertion happens.. therefore we do not need to have this check here, i.e. the result is always `true`.

Reply:
It should return name of namespace or `None`, I fixed that...